### PR TITLE
Add a janitorial 'borg plunger

### PR DESCRIPTION
**Pull request by lbnesquik**
## About The Pull Request
I got annoyed at not being able to plung vents when there's a clog. This PR corrects that. I copied all of the cyborg mop attributes to make this. This is my first PR on this server, but I did test it on a local server, and it indeed, plungs. 

## Why It's Good For The Game
More content for janitorial cyborg. They really should have a plunger, to be honest. It just makes sense to me. 

## Changelog
:cl:
add: Added a cyborg plunger for janitorial modules
/:cl:

### CODE REVIEW

**Comment by optimumtact** [comment]
this doesn't spawn by default on janitor borgs right?

**Comment by lbnesquik** [comment]
> this doesn't spawn by default on janitor borgs right?

No, it's not in the list of starter gear. 

image

File: code/game/objects/items/robot/robot_upgrades.dm [hunk 1]
```diff
@@ -208,6 +208,16 @@
 
 	items_to_add = list(/obj/item/cautery/prt)
 
+/obj/item/borg/upgrade/plunger
+	name = "janitor cyborg plating repair tool"
```
Comment (Sadboysuss) [inline code/game/objects/items/robot/robot_upgrades.dm:212]: Leftover from copying code

Comment (lbnesquik) [reply code/game/objects/items/robot/robot_upgrades.dm:212]: Thanks!

Comment (lbnesquik) [reply code/game/objects/items/robot/robot_upgrades.dm]: Fixed.

**Pull request merged**